scripts/metrics/codeformat.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Wrapper for executing "gradle spotlessJavaCheck". Make sure to execute
+# the script inside a task folder that contains a gradle project.
+# The return value of the gradle execution is saved in a file
+# "build/results/codeformat.yml".
+#
+# usage: codeformat.sh
+#
+# This script is step 5 in the Deploy-to-Grading pipeline that is
+# executed for every task that includes the codeformat metric.
+#
+
+DIR=build/results
```

Comment:
Diese Variablen tauchen ja auch in anderen Skripten immer wieder auf. Vermutlich brauchen wir mittelfristig nochmal eine Art Konfigurationshandling ...

Reply:
Das hier überschneidet sich auch im Python-Skript stark mit der Implementierung von der compile-Metrik. Da hatte ich auch schon überlegt, ob man irgendwann Funktionen in eine `utils.py` auslagern sollte.

Beim Konfigurationshandling im Shell-Skript fallen mir nur Umgebungsvariablen ein. Nicht, dass das irgendwann eine unübersichtliche Liste an Umgebungsvariablen wird.